Source/Csla/Serialization/Mobile/MobileFormatter.cs

```diff
@@ -117,6 +111,15 @@ private void ConvertEnumsToIntegers()
       }
     }
 
+    private MobileFormatterOptions GetOptions()
+    {
+      var cslaOptions = applicationContext.GetRequiredService<CslaOptions>();
+      var options = (MobileFormatterOptions)cslaOptions.SerializationOptions.FormatterOptions;
```

Comment:
Wouldn't be `cslaOptions.SerializationOptions.FormatterOptions as MobileFormatterOptions` be better?
With that you could react to an unexpected type instead of causing a cast exception here.